crates/sui-adapter-transactional-tests/tests/upgrade/remove_phantom.move

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//# init --addresses A0=0x0 A1=0x0 B0=0x0 B1=0x0 --accounts A
+
+//# publish --upgradeable --sender A
+module A0::base {
+    struct Foo<phantom T> { }
+}
+
+//# upgrade --package A0 --upgrade-capability 1,1 --sender A
+module A1::base {
+    struct Foo<T> { }
+}
+
+//# upgrade --package A0 --upgrade-capability 1,1 --sender A
+module A1::base {
+    struct Foo<T: store> { }
```

Comment:
You are also adding a `store` here, is that intentionaly?  That would be incompatible anyway but for another reason.

Reply:
Yep! But I just wanted to make sure we were covering that as well here.